### ISSUE TITLE
Add direct link to OONI Probe app apk

### ISSUE THREAD

**Issue by hellais** [TheTorProject/ooni-web#277]
To overcome the fact that in some countries there is no Play Store.

We can possibly use a direct link to the f-droid apk or to the apk in the github release.

**Comment by p1g30n** [comment]
Does F-Droid offer links that always point to the latest release? Github offers that but I fail to find built apks in the probe-android repo

**Comment by hellais** [comment]
Yeah there is a link to the f-droid apk here: https://f-droid.org/en/packages/org.openobservatory.ooniprobe/.

I think am not sure how to get the latest one automatically though, maybe @eighthave knows if this is possible.

**Comment by eighthave** [comment]
there is a Setting to enable automatic updates

**Comment by hellais** [comment]
@eighthave the question is if it's possible to get a direct link to the f-droid apk for the latest OONI Probe apk, something like: `https://f-droid.org/repo/org.openobservatory.ooniprobe_LATEST.apk`

**Comment by eighthave** [comment]
not currently, since we discourage direct downloads.  It is much safer
to install via F-Droid.

**Comment by eighthave** [comment]
F-Droid has the "Nearby Swap" for that purpose.  It also supports
mirrors on USB thumb drives and SDCards.

**Comment by hellais** [comment]
Right, though it requires that the user first installs f-droid and then the app, which adds some friction to the install process. It's probably a good idea to document somewhere this approach as the best practice, but we probably still want to offer also a simple direct apk download.

@eighthave is there a link to some f-droid documentation on how this feature works?

**Comment by p1g30n** [comment]
Seeing that built apks are now included in the android releases ( https://github.com/ooni/probe-android/releases/tag/v2.2.0 ) here is a propositional draft for displaying them on the ooni website: https://repl.it/repls/DullRoundTests

There seems to be no way to have a static link that always points to the latest apk, this approach is the best I can think of at the moment without having to add stuff server side.

**Comment by p1g30n** [comment]
> Seeing that built apks are now included in the android releases ( https://github.com/ooni/probe-android/releases/tag/v2.2.0 ) here is a propositional draft for displaying them on the ooni website: https://repl.it/repls/DullRoundTests
> 
> There seems to be no way to have a static link that always points to the latest apk, this approach is the best I can think of at the moment without having to add stuff server side.

Thinking about this more, solving it server side would probably be more elegant (and not rely on the user having Javascript enabled). I'm not familiar with probe-android's build process but maybe there's a way to add a build hook that always generates the link to the latest apk as a data template in Hugo?
